src/vs/workbench/browser/parts/titlebar/menubarControl.ts

```diff
@@ -65,7 +65,7 @@ export abstract class MenubarControl extends Disposable {
 		// 'Terminal': IMenu; {{SQL CARBON EDIT}} - Disable unusued menus
 		'Window'?: IMenu;
 		'Help': IMenu;
-		// [index: string]: IMenu | undefined; {{SQL CARBON EDIT}} - Disable unusued menus
+		[index: string]: IMenu | undefined;
```

Comment:
Why are we able to bring this back now?

Reply:
I don't know why it was removed in the first place, but it is needed for type checking. I can't see a reason we would want to remove it. I can check if there is something in the commit history for it.

Reply:
looks like it was changed https://github.com/microsoft/azuredatastudio/pull/4283 so who knows why it was changed (comments are important). I'll test.